packages/beacon-node/src/api/rest/base.ts

```diff
@@ -84,34 +84,34 @@ export class RestApiServer {
     // Log all incoming request to debug (before parsing). TODO: Should we hook latter in the lifecycle? https://www.fastify.io/docs/latest/Lifecycle/
     // Note: Must be an async method so fastify can continue the release lifecycle. Otherwise we must call done() or the request stalls
     server.addHook("onRequest", async (req, _res) => {
-      const {operationId} = req.routeConfig as RouteConfig;
-      this.logger.debug(`Req ${req.id as string} ${req.ip} ${operationId}`);
+      const operationId = req.routeSchema.operationId as string;
```

Comment:
It makes more sense to get this from schema as `operationId` is already part of standard type definition. We can drop including it in custom `routeConfig` altogether but will do that on the refactor branch